### DISCUSSION TITLE
TIL - 22/03/06

### DISCUSSION THREAD

**Discussion by leeheefull** [selab-hs/selab#108] [Check TIL]

**Comment by leeheefull** [comment]
### 작업 사항
* Fixture monkey를 적용하여 Board 테스트에 적용했다.
  * 랜덤 값 자동 바인딩이 가능해짐

### 이슈

**Comment by DongGeon0908** [comment]
### 작업 사항
- Auth 분리 작업 진행
- 미비된 Test 구현

### 이슈
- Server 구축 일정이 미뤄지고 있음
